examples/h2o/random_forest.ipynb

```diff
@@ -94,12 +94,13 @@
     "import mlflow\n",
     "import mlflow.h2o\n",
     "from h2o.estimators.random_forest import H2ORandomForestEstimator\n",
+    "\n",
```

Comment:
Can we revert the unrelated changes?

Reply:
Okay, I will revert those changes. Black autoformatted them.